src/app.js

```diff
@@ -1,20 +1,89 @@
-const express = require('express');
-const app = express();
+require('dotenv').config()
+require('express-async-errors')
+const express = require('express')
+const session = require('express-session')
+const app = express()
+const morgan = require('morgan')
+const bodyParser = require('body-parser')
 const cors = require('cors')
-const favicon = require('express-favicon');
-const logger = require('morgan');
+const favicon = require('express-favicon')
+const { xss } = require('express-xss-sanitizer')
+const helmet = require('helmet')
+const passport = require('passport')
+const mongoose = require('mongoose')
+mongoose.set('strictQuery', true)
+const cookieParser = require('cookie-parser')
+const rateLimit = require('express-rate-limit')
+const limiter = rateLimit({
+    windowMs: 15 * 60 * 1000, // 15 minutes
+    max: 60, // limit each IP to 100 requests per windowMs
+})
 
-const mainRouter = require('./routes/mainRouter.js');
+const logger = require('./logs/logger')
```

Comment:
@DYA13 , hmm, I just realized that your `logs` folder is outside of the `src` folder where the `app.js` lives. So you'll either need to update this to: `../logs/logger` (the two dots mean "the folder above/outside of this one); or move the `logs` folder to be inside the `src` folder together with `app.js`